planet/specs.py

```diff
@@ -48,8 +48,8 @@ def validate_bundle(bundle):
 
 
 def validate_item_type(item_type, bundle):
-    bundle = validate_bundle(bundle)
     supported = get_item_types(bundle)
+    bundle = validate_bundle(bundle)
```

Comment:
@kevinlacaille am I missing something or does this repeat the validation on line 117 of order_request.py?

Reply:
I wanted to make sure `validate_item_type()` had a validated bundle, but I suppose putting it after `get_item_type()` was useless haha. That being said, now when we call `get_item_type()` that'll raise a `KeyError` and raise `validate_bundle()`, which will raise the proper exception.